homeassistant/components/light/hue.py

```diff
@@ -51,6 +52,7 @@
     vol.Required(CONF_HOST): cv.string,
```

Comment:
Whilte testing your PR I realized that this one should actually be `vol.Optional`